### ISSUE TITLE
Modify getLatestZiti to have an optional parameter for OS

### ISSUE THREAD

**Issue by gberl002** [openziti/ziti#638]
When following the README for locally building the dockerfile the steps state to use getLatestZiti however, that function pulls the ziti specific to the user's OS when it should actually pull the Ubuntu version since it will be used in docker Ubuntu containers.

Changes needed:
Update getLatestZiti to allow forcing the OS type
Update the dockerfile README step to use this optional parameter

**Comment by dovholuknf** [comment]
doesn't it already detect this on your behalf? we just need to modify the detection to find 'arm v amd'?

**Comment by gberl002** [comment]
No, for instance, I'm using a Mac so when I call getLatestZiti it downloads the darwin version. The issue is that when I try to build the dockerfile, it will load the darwin executables which won't work in the ubuntu container when deployed.

**Comment by gberl002** [comment]
Fixed in https://github.com/openziti/ziti/pull/639 by updating the README to better explain how to build the docker image for various uses.
